### PR TITLE
Updated Makefile and DockerREADME.md, this should now work on windows

### PR DESCRIPTION
**Pull request by gdeak-unosquare**

### CODE REVIEW

File: BackEnd/DockerREADME.md [hunk 1]
```diff
@@ -2,33 +2,28 @@
 **With Docker**
 ```
 # to build the admin core
-docker build -f docker/admin.core.Dockerfile -t unosquare/admincore:0.0.1 .
-docker tag docker/admincore:0.0.1 docker/admincore:latest
+docker build -f docker/admin.core.Dockerfile -t unosquare/admincore:<version> .
+docker tag docker/admincore:<version> docker/admincore:latest
```
Comment (BrownMark) [inline BackEnd/DockerREADME.md]: I think this should be unosquare/admincore here.

**Review by BrownMark — CHANGES_REQUESTED** [review]
Just one minor comment

File: BackEnd/DockerREADME.md (hunk 1 above)
Comment (gdeak-unosquare) [reply BackEnd/DockerREADME.md]: yep, you are right

**Comment by gdeak-unosquare** [comment]
@BrownMark. I have fixed the typo

**Comment by BrownMark** [comment]
Looks good

**Review by BrownMark — APPROVED** [review]

**Pull request merged**